turingschool-examples/romcom
# Feature: Save and View Covers

# Pull Request Template

#### What’s this PR do? 
1. It adds unique saved covers as elements in the `savedCovers` array.
2. It adds unique saved covers as mini covers in the saved covers page.

#### Where should the reviewer start?
Start in `main.js` and review the saveCover() and formatSavedCovers() functions.

#### How should this be manually tested?
Generate new covers and save each new one. Try saving the same poster a couple times. Click over to [View Saved Covers] and check for nicely formatted, unique covers that you saved.

#### Any background context you want to provide? 
Noticed a bug in some displays unrelated to this iteration that we will need to fix. It should be as simple as adding and removing some hidden classes so that only the views we want to see are displayed.

#### Screenshots (if appropriate): 
![Screen Shot 2020-08-29 at 10 19 51 AM (2)](https://user-images.githubusercontent.com/59029768/91641401-50290380-e9e1-11ea-9371-7f96ff392c7d.png)
